app/knights.py

```diff
@@ -0,0 +1,38 @@
+class Knights:

```

Comment:
Rename this class. You create one Knight here, not Knights, so the name is wrong